app/services/calculate_candidate.rb

```diff
@@ -0,0 +1,11 @@
+class CalculateCandidate
+  def initialize(answers)
+    @answers = answers
+  end
+
+  def execute
+    answers_by_candidates = @answers.group_by { |a| a.candidate }
+    max = (answers_by_candidates.map { |k, v| v.count }).max
```

Comment:
Unused block argument - `k`. If it's necessary, use `_` or `_k` as an argument name to indicate that it won't be used.